README.md

```diff
@@ -5,6 +5,7 @@ XAML Behaviors is an easy-to-use means of adding common and reusable interactivi
 Getting Started
 -------------------
 **Where to get it**
+ - [![NuGet version](https://badge.fury.io/nu/Microsoft.Xaml.Behaviors.Wpf.svg)](https://badge.fury.io/nu/Microsoft.Xaml.Behaviors.Wpf)
```

Comment:
This is quite useful and I'm fine with accepting it, but I think I would prefer to use the [shields.io](https://shields.io) version instead, as that one the most commonly used by Microsoft projects (plus, it states that no tracking is done, but I can't find any information on that on badge.fury.io)

```
[![NuGet version](https://img.shields.io/nuget/v/Microsoft.Xaml.Behaviors.Wpf)](https://www.nuget.org/packages/Microsoft.Xaml.Behaviors.Wpf)
```

[![NuGet version](https://img.shields.io/nuget/v/Microsoft.Xaml.Behaviors.Wpf)](https://www.nuget.org/packages/Microsoft.Xaml.Behaviors.Wpf)